pkg/router/template/router.go

```diff
@@ -380,6 +381,41 @@ func (r *templateRouter) watchVolumeMountDir(path string, reloadFn func()) error
 	return nil
 }
 
+// watchVolumeMountDir adds a watcher on path, which should be a secret or
+// configmap volume mount, and calls reloadFn when a change is detected.
+func (r *templateRouter) watchDir(path string, reloadFn func()) error {
```

Comment:
```suggestion
// watchDir adds a watcher on path, which should be a secret or
// configmap volume mount, and calls reloadFn when a change is detected.
func (r *templateRouter) watchDir(path string, reloadFn func()) error {
```

Reply:
The comment and function name don't match.  (Did this get fixed earlier and then unfixed?)